### ISSUE TITLE
Change non-free icons

### ISSUE THREAD

**Issue by julianfairfax** [osmandapp/OsmAnd#13386]
# 🚀 Change non-free icons

### Description
OsmAnd currently uses some non-free icons: https://github.com/osmandapp/Osmand/blob/master/LICENSE#L52
![image](https://user-images.githubusercontent.com/43753131/146400555-8ae51dfc-7471-40a4-b2e2-483bca8c7d5c.png)

### Describe the solution you'd like

Replace these icons with freely licensed icons.

### Describe alternatives you've considered

Obviously there's nothing wrong with using the current ones, but there's also no point point in using 75 non-free icons when you can find or create freely licensed versions of those icons.

**Comment by julianfairfax** [comment]
(Of course there are very many more important issues, I know that and I'm not saying those shouldn't be focused on first.)

Since I haven't seen any activity on this for a bit, if I were to find freely licensed icons to replace those, would they be accepted in the form of a pull request, or do you have to be the ones to find those?

--

If you're seeing this and happen to know of where I should look for these freely licensed icons, please let me know on this issue, or personally.

**Comment by vshcherb** [comment]
I'm not sure icons 8 is used at all. needs to be checked

**Comment by julianfairfax** [comment]
> I'm not sure icons 8 is used at all. needs to be checked

I mean I don't know if the files in https://github.com/osmandapp/OsmAnd-resources/tree/master/icons/svg/icons are actually used, and I didn't check all of them. But blunderbuss for example is the same as musket on icons8.

**Comment by vshcherb** [comment]
Yes, https://github.com/osmandapp/OsmAnd-resources/tree/master/icons/svg/icons8 this one is present and we will be happy to delete which one are not used so we know what needs to be done. Though we need to look up in scripts like this 
https://github.com/osmandapp/OsmAnd-resources/blob/master/icons/tools/generatepng.sh
https://github.com/osmandapp/OsmAnd-resources/blob/master/icons/tools/sortfiles.sh

Which one to delete

**Comment by julianfairfax** [comment]
It seems, at least according to https://github.com/osmandapp/OsmAnd-resources/blob/master/icons/tools/sortfiles.sh, that the only icons used are [sail_boat](https://github.com/osmandapp/OsmAnd-resources/blob/master/icons/tools/sortfiles.sh#L1220), [user](https://github.com/osmandapp/OsmAnd-resources/blob/master/icons/tools/sortfiles.sh#L2072), [web_search](https://github.com/osmandapp/OsmAnd-resources/blob/master/icons/tools/sortfiles.sh#L2076), [wikipedia](https://github.com/osmandapp/OsmAnd-resources/blob/master/icons/tools/sortfiles.sh#L2074), and [worldwide_location](https://github.com/osmandapp/OsmAnd-resources/blob/master/icons/tools/sortfiles.sh#L2079).

**Comment by julianfairfax** [comment]
I've done some looking and it appears that sail_boat.svg is used in the categories search as the nautical icon. The boat icon from the boat profile could be used to replace it. It appears that wikipedia.svg is also used in the categories search. The Wikipedia icon from the map download page could be used to replace it.

I haven't looked everywhere, but I didn't see user.svg, web_search.svg, or worldwide_location.svg anywhere obvious. Is it possible they already aren't actually used? There are definitely already icons for search and locations in the app that could replace the last two, not sure about user.svg in any case.

**Comment by dmpr0** [comment]
Done https://github.com/osmandapp/OsmAnd-resources/commit/2405a2919624ef9dcc2c4bf0ea5d311e797fa072
